apps/native-component-list/src/screens/LocalModules/LocalModulesScreen.tsx

```diff
@@ -0,0 +1,5 @@
+import LocalModulesDemo from '../../../../bare-expo/localModulesExamples/LocalModulesDemoComponent';
```

Comment:
why not move the example from that arbitrary directory to here? That's how we intend ppl to use it anyways (deep in hierarchy)